gtests/net/mptcp/sockopts/sockopt_maxseg_tcp.pkt

```diff
@@ -0,0 +1,31 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0.0  socket(..., SOCK_STREAM, IPPROTO_TCP) = 3
++0.0  getsockopt(3, SOL_TCP, TCP_MAXSEG, [536], [4]) = 0
+
++0.0  setsockopt(3, SOL_TCP, TCP_MAXSEG, [1000], 4) = 0
++0.0  getsockopt(3, SOL_TCP, TCP_MAXSEG, [1000], [4]) = 0
+
++0.0  bind(3, ..., ...) = 0
++0.0  listen(3, 1) = 0
+
++0.0    <  S   0:0(0)          win 8000  <mss 1460, sackOK, nop, nop, nop, wscale 8>
++0.0    >  S.  0:0(0)  ack 1   win 65535 <mss 1000, nop, nop, sackOK, nop, wscale 8>
```

Comment:
please remove the `win` for all outgoing (`>`) packets: their value can vary depending on different settings, and it can change depending on the kernel version. It is optional for this direction (but mandatory for incoming ones `<`)

→ So please, also remove the `win` in line 28 as well